pilot/pkg/features/pilot.go

```diff
@@ -622,6 +622,9 @@ var (
 	CanonicalServiceForMeshExternalServiceEntry = env.RegisterBoolVar("LABEL_CANONICAL_SERVICES_FOR_MESH_EXTERNAL_SERVICE_ENTRIES", false,
 		"If enabled, metadata representing canonical services for ServiceEntry resources with a location of mesh_external will be populated"+
 			"in the cluster metadata for those endpoints.").Get()
+
+	LocalClusterSecretWatcher = env.RegisterBoolVar("LOCAL_CLUSTER_SECERT_WATCHER", false,
+		"If enabled, the cluster secret watcher will watch the namespace in which Istiod runs").Get()
```

Comment:
This description is confusing. Even if not enabled, the cluster secret watcher will normally "watch the namespace in which Istiod runs". It's only in the external istiod case, that by default, it will watch a different config cluster. Maybe something like this:
```suggestion
		"If enabled, the cluster secret watcher for an external Istiod will watch the namespace in which Istiod runs instead of in the config cluster").Get()
```